tuned.spec

```diff
@@ -403,6 +424,15 @@ if [ -d %{_sysconfdir}/grub.d ]; then
 fi
 
 
+%posttrans ppd
+# if power-profiles-daemon was active before installing tuned-ppd,
+# start tuned-ppd right away
+if [ -f %{_localstatedir}/lib/rpm-state/tuned/ppd-active ]; then
+  systemctl start tuned-ppd
+  rm -rf %{_localstatedir}/lib/rpm-state/tuned
```

Comment:
This will probably trigger rpmlint warning, thus it's better to do `rm -f` and then `rmdir`, but this is minor problem that can be ignored/waived or fixed later.